change/@fluentui-react-components-d196da53-af3d-47b0-aa8b-2ad653756f37.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Add v9 Icons to Storybook",
+  "packageName": "@fluentui/react-components",
+  "email": "ololubek@microsoft.com",
+  "dependentChangeType": "patch"
```

This should be `none` since it doesn't affect the published package.
```suggestion
  "type": "none",
  "comment": "Add v9 Icons to Storybook",
  "packageName": "@fluentui/react-components",
  "email": "ololubek@microsoft.com",
  "dependentChangeType": "none"
```